resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -168,6 +168,16 @@ private[spark] object Config extends Logging {
       .checkValues(ExecutorRollPolicy.values.map(_.toString))
       .createWithDefault(ExecutorRollPolicy.TOTAL_GC_TIME.toString)
 
+  val MINIMUM_TASKS_PER_EXECUTOR_BEFORE_ROLLING =
+    ConfigBuilder("spark.kubernetes.executor.minTasksPerExecutorBeforeRolling")
+      .doc("The minimum number of tasks per executor before rolling. " +
+        "Spark will not roll newly created executors whose total number of tasks is smaller" +
```

Comment:
nit: space at the end, also do we have to mention "newly created" here? what about "Spark will not roll executors whose total number of tasks is smaller"

Reply:
Thank you for your review, @sunchao . I'll address them both.